spec/index.html

```diff
@@ -427,14 +498,26 @@ <h2>RDF Documents and Syntaxes</h2>
     <em>RDF 1.2 Concepts and Abstract Syntax</em>,
     but can conform to such other specifications that normatively
     reference terms defined here.</p>
+
+  <p>This specification establishes two conformance levels:</p>
+
+  <ul>
+    <li><dfn class="no-export lint-ignore">Full</dfn> conformance
+      supports <a data-lt="RDF graph">graphs</a> and <a data-lt="RDF dataset">datasets</a>
+      containing <a>quoted triples</a>.
+      Concrete syntaxes in which such graphs and datasets can be expressed include
+      [[RDF12-N-TRIPLES]], [[RDF12-N-QUADS]], [[RDF12-TURTLE]], and [[RDF12-TRIG]].</li>
+    <li><dfn class="no-export lint-ignore">Classic</dfn> conformance
+      only supports <a data-lt="RDF graph">graphs</a> or <a data-lt="RDF dataset">datasets</a>,
+      that do not allow the use of use <a>quoted triples</a>.</li>
+  </ul>
```

Comment:
```suggestion
  </ul>
  <p class="ednote">The conformance levels described above are tentative,
    and still the subject of group discussion.
    As an alternative, "profiles" rather than conformance levels,
    may be abandoned, or described in another specification.</p>
```

Reply:
```suggestion
  </ul>
  <p class="ednote">The conformance levels described above are tentative,
    and still the subject of group discussion. An alternative to conformance
    levels, "profiles", may be adopted instead, abandoned, or described in 
    another specification.</p>
```